guides/common/modules/proc_configuring-server-to-provision-secure-boot-enabled-hosts.adoc

```diff
@@ -0,0 +1,84 @@
+[id="Configuring-{project-context}-Server-to-Provision-{client-os-context}-on-Secure-Boot-Enabled-Hosts_{context}"]
+= Configuring {ProjectServerTitle} to Provision {client-os} on Secure Boot Enabled Hosts
+
+Secure Boot follows a chain of trust from the start of the host to the loading of Linux kernel modules.
+The first shim that is loaded determines which distribution is allowed to be booted or kexec'ed until next reboot.
```

Comment:
Unsure if "kexec'ed" is allowed by IBM style guide. I am OK, any concerns? @asteflova ?

Reply:
IBM style guide is not that detailed but it generally does not allow turning commands into verbs.

Maybe this would work: "The first shim that is loaded determines which distribution can be booted or loaded by using a `kexec` system call until the next reboot."

Reply:
IMO your suggestion would fit, @apinnick.

What do you think, @jloeser?

Reply:
Would be OK.

Reply:
Reworded